### PR TITLE
API-3760/veteran crisis line test coverage

### PR DESCRIPTION
**Pull request by johanna-adhoc**
### [API-3760](https://vajira.max.gov/browse/API-3760)
Added test coverage for components VeteransCrisisLinePanel and VeteransCrisisLine
Screen Shot 2020-12-08 at 12 46 30 PM

### Process
- [X] Tests added or updated for change
- [N/A] Docs updated
  - [N/A] If you have made structural changes to the site, the [Developer Portal Content Types](https://community.max.gov/x/qUwOg) docs have been updated
  - [N/A] If you have made/confirmed any decisions about user interactions and accessibility, the [Accessible Component Design](https://community.max.gov/x/xS8mgg) docs have been updated
- [N/A] Accessibility concerns have been considered and addressed
  
  - [N/A] If you've made any UI changes, you've run an axe check using the axe browser extension that reported no new issues

### CODE REVIEW

File: src/components/crisisLine/VeteransCrisisLine.test.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import * as React from 'react';
+
+import 'jest';
+import '@testing-library/jest-dom/extend-expect';
+import {  fireEvent, render, waitFor } from '@testing-library/react';
```
Comment (anthony-kopczyk) [inline src/components/crisisLine/VeteransCrisisLine.test.tsx:5]: extra spaces

Comment (anthony-kopczyk) [inline src/components/crisisLine/VeteransCrisisLine.test.tsx:5]: extra spaces

File: src/components/crisisLine/VeteransCrisisLinePanel.test.tsx [hunk 2]
```diff
@@ -0,0 +1,29 @@
+import * as React from 'react';
+
+import 'jest';
+import '@testing-library/jest-dom/extend-expect';
+import {   render } from '@testing-library/react';
```
Comment (anthony-kopczyk) [inline src/components/crisisLine/VeteransCrisisLinePanel.test.tsx:5]: extra spaces

File: src/components/crisisLine/VeteransCrisisLinePanel.test.tsx [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+
+import VeteransCrisisLinePanel from './VeteransCrisisLinePanel';
+
+describe('VeteransCrisisLine', () => {
+  it('checks that panel displays with correct information.', () => {
+    const {  getByRole, queryByText, queryAllByRole } = render(<VeteransCrisisLinePanel  />);
+    const title = getByRole('heading');
+    expect(title).toHaveTextContent("We're here anytime, day or night - 24/7");
+    expect(queryByText('If you are a Veteran in crisis or concerned about one, connect with our caring, qualified responders for confidential help. Many of them are Veterans themselves.')).toBeInTheDocument();
```
Comment (anthony-kopczyk) [inline src/components/crisisLine/VeteransCrisisLinePanel.test.tsx:14]: My auto-format switches this into multiple lines.

File: src/components/crisisLine/VeteransCrisisLine.test.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  it('checks that modal button displays with correct information.', () => {
+    const { queryByRole } = render(<VeteransCrisisLine  />);
+    const modalButton = queryByRole('button');
+
+    expect(modalButton).toBeInTheDocument();
+    expect(modalButton).toHaveTextContent('Talk to the Veterans Crisis Line now');
+    expect(modalButton).toHaveAttribute('data-show', '#crisis-line-modal');
+  });
+
+  it('checks open/close functionality of dialog works correctly.', async() => {
```
Comment (anthony-kopczyk) [inline src/components/crisisLine/VeteransCrisisLine.test.tsx:19]: My auto-formatter puts a space between `async` and `()`.

**Review by anthony-kopczyk — COMMENTED** [review]
The linter didn't throw any errors for me on this, but my auto-format takes out some spaces from a few spots in the file. I commented the most notable spots. Just commenting for now until I hear back on this. If it's something with our IDE formatters being out of sync then I should be able to approve because it won't cause an issue with the build pipeline.

File: src/components/crisisLine/VeteransCrisisLine.test.tsx (hunk 4 above)
Comment (johanna-adhoc) [reply src/components/crisisLine/VeteransCrisisLine.test.tsx]: I tried formatting with prettier... hopefully that handled the issues?

Comment (anthony-kopczyk) [reply src/components/crisisLine/VeteransCrisisLine.test.tsx]: It looks good to me now! 👍

**Review by anthony-kopczyk — APPROVED** [review]
Looks good with the spacing commit

**Review by anthony-kopczyk — APPROVED** [review]
Looks good to me!

**Review by TS000 — APPROVED** [review]
Looks great!

**Pull request merged**